components/dashboard_modules/ExpectedOutput.tsx

```diff
@@ -0,0 +1,43 @@
+import NumberDisplay from './NumberDisplay';
+import { formatter, useFutureGraphData } from 'lib/utils';
+
+interface ForecastDataPoint {
+  target_datetime_utc: number;
+  expected_generation_kw: number;
+}
+
+const computeMipointRiemmanSum = (points: ForecastDataPoint[] | undefined) => {
```

Comment:
Typo: "Riemman" &rarr; "Riemann". Though, in my opinion, this function's name leaks its implementation too much. I suggest a rename to a more general definition (in case we want to change impl) "getTotalExpectedOutput".
```suggestion
const computeMipointRiemannSum = (points: ForecastDataPoint[] | undefined) => {
```